source/game/kart/KartMove.cc

```diff
@@ -1318,7 +1305,7 @@ void KartMove::calcStandstillBoostRot() {
         }
     }
 
-    if (m_bWallBounce) {
+    if (m_flags.onBit(eFlags::WallBounce)) {
```

Comment:
Hello Sharks. Today, I would like to pitch to you an innovate quality-of-life invention. Introducing:

`bool EGG::TBitFlag::operator()(E e)` to allow for short-hand call to onBit

Reply:
This creates ambiguity with the constructor call and implied `operator T()`.